src/pages/About.vue

```diff
@@ -29,6 +152,29 @@ export default {
   },
   components: {
     History
+  },
+  data() {
+    return {
+      servicesActiveImage: 0,
+      servicesImages: [
+        "/assets/img/cloud.jpg",
+        "/assets/img/unix-shell.jpg",
+        "/assets/img/printer.jpg"
+      ]
+    };
   }
 };
 </script>
+<style scoped>
+p.card {
+  transform: none;
+  transition: transform 0.6s;
+  width: 100%;
+}
+.active-card {
+  transform: translate(1.75em, 0) !important;
```

Comment:
Woah, `!important` is pretty dangerous to use. Not sure why it's here.

Reply:
It's probably from when I was just messing around with trying to get stuff to work.

I can remove it and it should still work now that I've got the implementation sorted out

Reply:
You probably just need a `translateX` instead of `translate`